server/passport/index.js

```diff
@@ -0,0 +1,74 @@
+const bcrypt = require("bcryptjs");
+const passport = require("passport");
+const LocalStrategy = require("passport-local").Strategy;
+const session = require("express-session");
+const { Retailer } = require("../models");
+
+const saltRounds = 12;
+
+passport.use(
+  "login",
+  new LocalStrategy(
+    {
+      usernameField: "username",
+      passwordField: "password"
+    },
+    (username, password, done) => {
+      try {
+        Retailer.findOne({ username: username })
```

Comment:
please add `.select("password")` to the end of this line to explicitly add the password to the return.